### ISSUE TITLE
Session biases

### ISSUE THREAD

**Issue by GalKepler** [PennLINC/qsiprep#410]
I've encountered some troubling issues when working with a longitudinal dataset...
This dataset includes two sessions per participant, each including a DWI series (and another B0 series with an opposite phase encoding direction).

Visual inspection of the results of the preprocessing looks promising, but upon deeper inspection, I found some issues:
1. There's a session bias. It means that the overall signal in one session is higher than in the other.
2. The same bias appears when reconstructing the diffusion tensor... for example, MD appears to be higher throughout the entire brain when derived from one session's DWI in comparison to the other.
3. Just to make sure it didn't have anything to do with the reconstruction method, I've checked this effect using the tensor, kurtosis, and RESTORE models. The effect remains...
4. It seems that the later session is the one with the overall higher signal (and MD).

To make sure that this effect originates from the preprocessing pipeline, I executed my own pipeline on the same data and this effect didn't occur.

I would love your input on this one (:

**Comment by GalBenZvi** [comment]
No, should I give it a try? Won't it skip the intensity scaling if I do?

**Comment by mattcieslak** [comment]
Did you ever figure this out? I wonder, was your other pipeline setting a mean value in the dwis?
